cities/crud.py

```diff
@@ -0,0 +1,55 @@
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from cities import models, schemas
+
+
+async def create_city(
+    db: AsyncSession,
+    city: schemas.CityCreate
+) -> models.City:
+    db_city = models.City(name=city.name, additional_info=city.additional_info)
+    db.add(db_city)
+    await db.commit()
+    await db.refresh(db_city)
+    return db_city
+
+
+async def get_cities(db: AsyncSession) -> list[models.City]:
```

Comment:
The return type hint `list[models.City]` requires Python 3.9 or later. If you're using an earlier version, consider using `List[models.City]` from the `typing` module.